geekhours/command.py

```diff
@@ -33,3 +40,31 @@ def remove_course(self, arg: str):
     def remove_donelist(self, date: str, course: str):
         """ Call database.remove_donelist() """
         self.database.remove_donelist(date, course)
+
+    @staticmethod
+    def dump_to_csv(records: str, csvfile: str, fields: Tuple):
+        """ dump outputs to csv format file.
+
+        args:
+            records: Target data to dump.
+            csvfile: File path to save.
+            fields: Tuple object of a header row of the records.
+        """
+
+        with open(csvfile, 'w', newline='') as outcsv:
```

Comment:
It should be added a check if file exists before open(). If exists, the method should return.

Reply:
Add `raise FileExistsError` if file already exists.